public/app/features/explore/LiveTailButton.tsx

```diff
@@ -97,21 +97,19 @@ export function LiveTailButton(props: LiveTailButtonProps) {
   const theme = useTheme();
   const styles = getStyles(theme);
   const buttonVariant = isLive && !isPaused ? 'active' : 'default';
+
   const onClickMain = isLive ? (isPaused ? resume : pause) : start;
 
   return (
     <ButtonGroup>
-      <Tooltip
-        content={isLive && !isPaused ? <>Pause the live stream</> : <>Start live stream your logs</>}
-        placement="bottom"
-      >
+      <Tooltip content={isLive ? <>Pause the live stream</> : <>Live stream your logs</>} placement="bottom">
         <ToolbarButton
           iconOnly={splitted}
           variant={buttonVariant}
-          icon={!isLive || isPaused ? 'play' : 'pause'}
+          icon={!isLive ? 'play' : 'pause'}
           onClick={onClickMain}
         >
-          {isLive && isPaused ? 'Paused' : 'Live'}
+          {isPaused ? 'Live' : 'Paused'}
```

Comment:
Should this be the other way around or maybe this is the call to action?
```suggestion
          {isPaused ? 'Paused' : 'Live'}
```

Reply:
ah, good catch. Wrong merge when rebasing this, fixed this in another PR.